app-tasks/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/azavea/openjdk-gdal:2.3-jdk8-slim
+FROM quay.io/azavea/openjdk-gdal:2.4-jdk11-slim
```

Comment:
This is going to require testing of an upload to ensure that the GDAL upgrade won't cause problems.

Reply:
I was able to upload `cog-pa_m_4208064_se_17_1_20150727.tif` with the `4-band PlanetScope` datasource:

![image](https://user-images.githubusercontent.com/1774125/59777136-13606900-9282-11e9-9611-a16c4a812eda.png)


![image](https://user-images.githubusercontent.com/1774125/59777125-0cd1f180-9282-11e9-97a7-db959e063adf.png)